src/terrain/routes/schemas/vice.clj

```diff
@@ -127,4 +127,21 @@
    (optional-key :app-name)      (describe (maybe String) "The name of the running app for the analysis")
    (optional-key :external-id)   (describe (maybe UUID) "The value of the external_id field in the job_steps table. Used widely in the API")
    (optional-key :user-id)       (describe (maybe UUID) "The UUID assigned to the user that launched the analysis")
-   (optional-key :username)      (describe (maybe String) "The user of the user that launched the analysis")})
\ No newline at end of file
+   (optional-key :username)      (describe (maybe String) "The user of the user that launched the analysis")})
+
+(defschema NonAdminFilterParams
+  (dissoc FilterParams (optional-key :user-id) (optional-key :username)))
```

Comment:
There's no need to change this code, but [metosin/schema-tools](https://cljdoc.org/d/metosin/schema-tools/0.12.2/api/schema-tools.core) has some really handy functions for dealing with this.